stories/settings.stories.js

```diff
@@ -91,8 +92,11 @@ storiesOf( 'Settings', module )
 		global._googlesitekitLegacyData.canAdsRun = true;
 		global._googlesitekitLegacyData.modules.analytics.setupComplete = false;
 		global._googlesitekitLegacyData.modules.adsense.active = false;
+		const setupRegistry = ( { dispatch } ) => {
+			dispatch( CORE_MODULES ).registerModule( 'adsense' );
+		};
 		return (
-			<WithTestRegistry>
+			<WithTestRegistry callback={ setupRegistry }>
```

Comment:
I think we'll need to modify the `setupRegistry` implementation here to set up modules correctly in the datastore, using the `provideModules` function, passing the actual dependencies that these modules have.

The current story here right now does not reflect what the same UI actually looks like in the product - the store state here is basically the loading state, which we shouldn't use. Outside of loading, it shouldn't be possible for a module to show up with the grayed-out, disabled UI while not having a `ModuleSettingsWarning`. I believe the reason this happens is because the datastore state here isn't set up at all (except the AdSense module registration), which means that `canActivateModule` will return `undefined`, causing the area to be grayed-out - while at the same time there is no error.

Reply:
I've gone through and added the modules to the stories test with https://github.com/google/site-kit-wp/commit/97e937d57b49991d18b9f7f86426dce009c3b1e3#diff-49d6554e5e787508efbc6a4b5dc1910eb74664f8ad5fa2c81226b7f811b7bbd4R96 and updated screenshots for a few of the tests in https://github.com/google/site-kit-wp/commit/764fece807fb977ca13c1818b3df4f3a667922ee

Reply:
The storybook completes fine, but I also noticed that in `Notifications > Module Setup Complete`, we're getting some errors thrown as we're using `provideModules` there too, but not a corresponding `registerModule`, which means no corresponding `checkRequirements()` on the module.

Reply:
@ivankruchkoff we still need to add the default `checkRequirements` to [`moduleDefaults`](https://github.com/google/site-kit-wp/pull/2335/files#diff-51079bbb133e20fa794c0e1fd5fd30b551dda818e0cfe0ba4162502670cf133a) in `assets/js/googlesitekit/modules/datastore/modules.js`.